cmd/describe/ingress/cmd.go

```diff
@@ -0,0 +1,87 @@
+package ingress
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+
```

Comment:
looks like `fmt-imports` might need to be ran

Reply:
huh, it didn't do anything :thinking: